frame/nomination-pools/src/lib.rs

```diff
@@ -2311,7 +2601,28 @@ impl<T: Config> Pallet<T> {
 		member.last_recorded_reward_counter = current_reward_counter;
 		reward_pool.register_claimed_reward(pending_rewards);
 
-		// Transfer payout to the member.
+		// Gets the commission percentage and payee to be paid if commission has
+		// been set. Otherwise, a zero commission and payee of `None` is returned.
+		let commission_current = &bonded_pool.commission.get_commission_and_payee(&pending_rewards);
```

Comment:
```suggestion
		let operator_share = &bonded_pool.commission.get_commission_and_payee(&pending_rewards);
```

better name? the word `current` felt quite out of context there.

Reply:
`maybe_commission` ?